app/views/trends/index.html.erb

```diff
@@ -2,7 +2,7 @@
 <% title _("Trends") %>
 <% title_actions new_link(_("Add Trend Counter")),
                  documentation_button('4.1.3Trends') %>
-
+<!-- TODO: Replace with either just out of sync or origin specific interval -->
```

Comment:
IMHO new setting "trends interval" would be suitable here, if people don't want to add additional settings, out of sync makes most sense to me